charts/nr-k8s-otel-collector/templates/daemonset-configmap.yaml

```diff
@@ -0,0 +1,289 @@
+apiVersion: v1 
+kind: ConfigMap
+metadata:
+  name: {{ include "newrelic.common.naming.fullname" . }}-daemonset-config
+  namespace: {{ .Release.Namespace }}
+  labels:
+    {{- include "newrelic.common.labels" . | nindent 4 }}
+data:
+  otel-collector-config.yaml: |
+    receivers:
+      otlp:
+        protocols:
+          grpc:
+          http:
+      hostmetrics:  
```

Comment:
We might want to sync with @newrelic/caos to understand the metrics they are planning to scrape

Reply:
usually we try to send the very same metrics so that the experience is similar. Moreover, they can save you some time, since they have done already some research

Reply:
I actually referenced a lot of the hostmetrics settings from NRDOT when I was pointed there by david gay, https://github.com/newrelic/opentelemetry-collector-releases/blob/main/configs/nr-otel-collector-agent-linux.yaml 
is this what you mean by the CAOS team?

Reply:
yes!